src/components/dashboard/main-content/course-enrollments/CourseEnrollments.jsx

```diff
@@ -128,24 +148,11 @@ const CourseEnrollments = ({ children }) => {
             })}
             isAcknowledgingAssignments={isAcknowledgingAssignments}
           />
-          {shouldShowMarkSavedForLaterCourseSuccess && (
-            <CourseEnrollmentsAlert variant="success" onClose={() => setShouldShowMarkSavedForLaterCourseSuccess(false)}>
-              <FormattedMessage
-                id="enterprise.dashboard.course.enrollment.saved.for.later.alert.text"
-                defaultMessage="Your course was saved for later."
-                description="Message when a course is saved for later."
-              />
-            </CourseEnrollmentsAlert>
-          )}
-          {shouldShowMoveToInProgressCourseSuccess && (
-            <CourseEnrollmentsAlert variant="success" onClose={() => setShouldShowMoveToInProgressCourseSuccess(false)}>
-              <FormattedMessage
-                id="enterprise.dashboard.course.enrollment.moved.to.progress.alert.text"
-                defaultMessage="Your course was moved to In Progress."
-                description="Message when a course is moved to In Progress."
-              />
-            </CourseEnrollmentsAlert>
-          )}
```

Comment:
[inform] These alerts are currently rendered under the `features.FEATURE_ENABLE_TOP_DOWN_ASSIGNMENT` feature flag when they are not related to assignments.